pkg/blockatlas/platform.go

```diff
@@ -65,6 +65,14 @@ type (
 		Lookup(coins []uint64, name string) ([]Resolved, error)
 	}
 
+	// LendingAPI Lending provider interface
+	LendingAPI interface {
+		Name() string
+		GetProviderInfo() (LendingProvider, error)
+		GetCurrentLendingRates(assets []string) ([]LendingAssetRates, error)
+		GetAccountLendingContracts(req AccountRequest) (*[]AccountLendingContracts, error)
```

Comment:
*[]AccountLendingContracts
what is that?

Reply:
my bad, sorry, fixed